bridges/evm/contracts/CacheBridge.sol

```diff
@@ -60,4 +60,29 @@ contract CacheBridge is Bridge, ICacheBridge {
 
         requestsCache[requestKey] = res;
     }
+
+    /// Performs oracle state relay and many times of oracle data verification in one go.
+    /// After that, the results which is an array of Packet will be recorded to the state by using the hash of RequestPacket as key.
+    /// @param _data The encoded data for oracle state relay and an array of data verification.
+    function relayMulti(bytes calldata _data) external {
```

Comment:
Add this function in `ICacheBridge` interface

Reply:
done